recipe/meta.yaml

```diff
@@ -7,6 +7,11 @@
 # ensure dagmc is defined (needed for conda-smithy recipe-lint)
 {% set dagmc = dagmc or 'nodagmc' %}
 
+{% if dagmc == 'dagmc' %}
+# prioritize dagmc variant via build number
+{% set build = build + 100 %}
```

Comment:
Does this prioritize dagmc? Is that what the community wants?

Reply:
Is there really any downside to having the dagmc be the default?

Reply:
No! I am definitely in favor, but just want to make sure it's a choice that's arrived at with intention.

Reply:
Normally I don't like to force extra dependencies on users, but in this case the hard work of getting DAGMC built and linked properly has already been done so I don't see any reason why we shouldn't include it by default.